discord/gateway.py

```diff
@@ -876,7 +884,7 @@ async def from_connection_state(
         hook: Optional[Callable[..., Coroutine[Any, Any, Any]]] = None,
     ) -> Self:
         """Creates a voice websocket for the :class:`VoiceClient`."""
-        gateway = f'wss://{state.endpoint}/?v=4'
+        gateway = f'wss://{state.endpoint}/?v=8'
```

Comment:
the main feature of v8 is buffered resuming, but this pr does not actually implement resuming